tests/generic_config_updater/test_cacl.py

```diff
@@ -94,6 +95,7 @@ def expect_acl_table_match(duthost, table_name, expected_content_list):
 def expect_res_success_acl_rule(duthost, expected_content_list, unexpected_content_list):
     """Check if acl rule added as expected
     """
+    time.sleep(1) # Sleep 1 sec to ensure caclmgrd does update in case of its UPDATE_DELAY_SECS 0.5s
```

Comment:
This is a short solution. And we need to understand if this is a feature bug. Please remove it after feature fix.

Reply:
Sure. Thanks